docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,14 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+* Learn how to integrate with Eclipse:
+    * [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+    * [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+    * [[Import an existing sbt build into Eclipse|EclipseSbt]]
```

Comment:
Yeah, I was just thinking about the symmetry of it: I see instructions for importing a Maven project, creating a new Maven project, importing an sbt project... what about creating a new sbt project? Even if it can't be done directly in Eclipse, if I were an Eclipse user who didn't know that, I'd look at that list and wonder what to do.